### PR TITLE
no expressions to default build args

### PR DESCRIPTION
**Pull request by nilsnolde**
really sorry.. I forgot to _not_ set the build arg for a docker build locally before merging.. now it'll work, ffs!

### CODE REVIEW

**Comment by nilsnolde** [comment]
made it a pr but canceled all ci jobs manually

**Comment by nilsnolde** [comment]
so a fresh build (without cache) takes 19 mins on the `release` job and 5 mins on the `debug` job. any idea why? there's not much difference between the 2 builds and I'd have expected a release build to be quicker. it can't be only the `LOGGING_LEVEL=TRACE` option right?

**Comment by kevinkreiser** [comment]
the release build has: `-DENABLE_SANITIZERS=ON` which will be the biggest factor in my opinion

File: Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN pip install --upgrade conan
 # configure the build with symbols turned on so that crashes can be triaged
 WORKDIR /usr/local/src/valhalla/build
 RUN cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_C_COMPILER=gcc
-RUN make all -j${CONCURRENCY}
+RUN make all -j${CONCURRENCY:-$(nproc)}
```
Comment (kevinkreiser) [inline Dockerfile:29]: i was wondering why you didnt do it like this in the previous one but couldnt be bothered to check docker to see what would happen

**Review by kevinkreiser — APPROVED** [review]

**Comment by nilsnolde** [comment]
🤞

**Pull request merged**